wrappers/nixos.nix

```diff
@@ -4,16 +4,10 @@ self:
   config,
   lib,
   ...
-}@args:
```

Comment:
I'm not as familiar with the language enough to know if this will have unexpected results @MattSturgeon

Reply:
I've been meaning to remove that every time I open the file with nixd running... The cleanup was never related to what I was working on though.

`args` is literally just a param/variable being defined here. If it's unused, its safe to delete. No hidden black magic.